lib/Swagger/src/Liftigniter/Metadata/Api/GeosApi.php

```diff
@@ -171,15 +171,15 @@ public function getGeosWithHttpInfo()
             );
 
             return array($this->apiClient->getSerializer()->deserialize($response, '\Swagger\Client\Liftigniter\Metadata\Models\Geo[]', $httpHeader), $statusCode, $httpHeader);
-        } catch (ApiException $apiException) {
-            switch ($apiException->getCode()) {
+        } catch (ApiException $e) {
```

Comment:
can you rename this to `$error` / `$response` or thx?

Reply:
@xkxd  I can but I don't want to :p This is autogenerated code